observation/rfi_scan.py

```diff
@@ -19,8 +19,9 @@
 # Add experiment-specific options
 parser.add_option('-m', '--min-duration', type="float", default=None,
                   help="The The minimum time to repeat the rfi scan over (default=%default)")
-parser.remove_option('-f')
-parser.remove_option('-p')
+parser.add_option('--long', type="float", default=None,
```

Comment:
Why not change `--long` to `--scan-duration` with a default value of 180? You'll get the same behaviour but things will be clearer. To me, `--long` sounds like a boolean and not something that expects a duration.